app/store/async-storage-wrapper.js

```diff
@@ -1,34 +1,49 @@
 import ReadOnlyNetworkStore from '../util/test/network-store';
 import AsyncStorage from '@react-native-async-storage/async-storage';
 import { isE2E } from '../util/test/utils';
+import { MMKV } from 'react-native-mmkv';
 
 /**
  * Wrapper class for AsyncStorage.
+ * (Will want to eventuall re-name since no longer async once migratted to mmkv)
  */
 class AsyncStorageWrapper {
   constructor() {
     /**
      * The underlying storage implementation.
      * Use `ReadOnlyNetworkStore` in test mode otherwise use `AsyncStorage`.
      */
-    this.storage = isE2E ? ReadOnlyNetworkStore : AsyncStorage;
+    this.storage = isE2E ? ReadOnlyNetworkStore : new MMKV();
   }
 
   async getItem(key) {
     try {
-      return await this.storage.getItem(key);
+      // asyncStorage returns null for no value
+      // mmkv returns undefined for no value
+      // therefore must return null if no value is found
+      // to keep app behavior consistent
+      let value = (await this.storage.getString(key)) || null;
```

Comment:
Most likely we do not have any value like this on a key: value, but maybe be safer here, instead of the operator `||`  that also checks for falsy values like 0, "", false, null and undefined, we would most likely want to use `??` that only fallback if it's null or undefined

```suggestion
      let value = (await this.storage.getString(key)) ?? null;
```